src/components/Sum.jsx

```diff
@@ -1 +1,9 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum({ a = 0, b = 0 }) {
```

Comment:
Use a named export for the Sum component using the arrow function like: `export const Sum = ...` instead of `export function Sum(...)`.